config/Shared/config_default-docker.dev.php

```diff
@@ -181,6 +181,7 @@
 $messageBrokerTransport = getenv('SPRYKER_TESTING_ENABLED') ? 'in-memory' : MessageBrokerAwsConfig::SQS_TRANSPORT;
 
 $config[MessageBrokerConstants::CHANNEL_TO_TRANSPORT_MAP] =
+$config[MessageBrokerAwsConstants::CHANNEL_TO_SENDER_TRANSPORT_MAP] =
```

Comment:
This is not correct at all, even for development we're using different transport for sending/receiving. Let's remove this line.